storage/src/main/java/io/delta/storage/HDFSLogStore.java

```diff
@@ -18,6 +18,9 @@
 
 import java.io.FileNotFoundException;
 import java.io.IOException;
+import java.io.InterruptedIOException;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
```

Comment:
nit: remove these two unused imports

Reply:
Method is used, but I'll remove the other one